### DISCUSSION TITLE
Uninterruptable animations like consumables and emotes?

### DISCUSSION THREAD

**Discussion by xkamronx** [Sixze/ALS-Refactored#415] [Q&A]
Just curious of the best solution for this type of use case; an animation that once it starts playing shouldn't be interrupted, in a way that doesn't break anything after it's done.

Extra addition is going to be adding a way to cancel them but not instantly, if that makes sense.

Has anyone done this on Refactored?

**Comment by ameaninglessname** [comment]
You can try to add a montage slot node (with new slot name) right after the `PostLocomotion` slot and play montage at that slot.

I haven't tried it, but the idea is to provide a more prioritized slot that is only used for animations that you need to play, so it can't be interrupted by montages with lower priority.
